tests/fast-integration/eventing-test/README.md

```diff
@@ -3,7 +3,10 @@
 
 ## Overview 
 
-This test covers the end-to-end flow for Eventing.
+This test covers the end-to-end flow for Eventing. It is divided in three parts:
```

Comment:
These filenames don't seem to match the name of the three files in this PR. Plus, I suggest you remove update from the name of the actual files. There is no update happing here.

Reply:
renamed